api/policies/hasGameName.js

```diff
@@ -10,10 +10,12 @@ module.exports = function (req, res, next) {
   if (req.body.hasOwnProperty('gameName')) {
     if (typeof req.body.gameName === 'string') {
       // Could add restrictions on allowable game names
-      if (req.body.gameName.length > 0) {
-        return next();
+      if (req.body.gameName.length < 0) {
```

Comment:
This isn't equivalent to the previous version. An empty string will have a length of 0 and not hit this
```suggestion
      if (!req.body.gameName.length) {
```